app/views/shared/_nav.html.erb

```diff
@@ -1,37 +1,33 @@
-<nav class="navbar navbar-expand-sm navbar-dark bg-dark">
-  <%= link_to "Oddio", landmarks_path, class: "navbar-brand" %>
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample03" aria-controls="navbarsExample03" aria-expanded="false" aria-label="Toggle navigation">
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
The nav bar is very messed up for me on mobile... I'd rather not make these changes. I already setup bootstrap for all that.

Reply:
I'm not attached to the changes I've made to the navbar.

Reply:
The navbar is better when I removed the html stuff out of the view.

Reply:
Nav bar should be working properly with the removal of the html syntax out of the view.